### DISCUSSION TITLE
NSPanelTools app starting when device rebooted and not autostarting HA

### DISCUSSION THREAD

**Discussion by rb0135** [seaky/nspanel_pro_tools_apk#48] [Q&A]
Hi All,

I have installed lastest version of the NSPanelTools (v2.0.0) and configured it (hopefully correctly). Ihave 2 "issues/questions" on the behaviour of it.

1) NSPanelsTools is displayed when booting device: The Tools App starts and is displayed. I have the button on the top of the app "Active" on, but I do not have the option "Resume On Boot" under setup trned on. This is Off

2) Under TOOLS, I setup "Launch App" and I have the Home Assistant Companion App listed. This doesnt seem to start due to #1 occurring. I also notice on the bottom of the Tools Screen, next to Setup, the HA icon is greyed out, so not sure what this meant for (I thought I read somewhere there is an HA ingetration for these tools, but have gone thar far yet). If I exit out og the Tools (using the < button on the bottom of the status bar) then HA companion starts

Is this the normal operation or have I maybe misconfigured something?

Thanks.

**Comment by seaky** [comment]
fixed in v2.0.1

**Reply by rb0135** [reply]
Excellent... Thanks

**Comment by seaky** [comment]
the HA is not available yet will be in v2.1
